packages/core/src/database/database.js

```diff
@@ -56,6 +55,26 @@ class Database {
     });
   }
 
+  serialize() {
+    return this.db.serialize();
+  }
+
+  async restore(serializedDb, forced) {
+    return new Promise((resolve, reject) => {
+      const collections = this.db.listCollections();
+      if (!collections.length || forced) {
```

Comment:
Just in case collections is null?
```suggestion
      if (!collections?.length || forced) {
```